src/thunderbots/software/ai/world/game_state.h

```diff
@@ -0,0 +1,124 @@
+#pragma once
+
+#include <geom/point.h>
+#include <util/refbox_constants.h>
+
+/**
+ * @brief Holds the state of the game according to the referee
+ *
+ * @details Contains information on what period of the game it is, what type of
+ * play to run, team scores, time remaining, etc.  During normal gameplay, the
+ * information in this class is received from the [ssl
+ * refbox](https://github.com/RoboCup-SSL/ssl-refbox)
+ * program over the network.
+ *
+ * Taken from
```

Comment:
Looks like part of this comment line is missing?

Reply:
fixed